src/token/ERC20MultiVotes.sol

```diff
@@ -200,18 +200,26 @@ abstract contract ERC20MultiVotes is ERC20, Auth {
     }
 
     /**
-     * @notice Transfer `amount` votes from the sender's delegation of `oldDelegatee` to `newDelegatee`.
-     * @param oldDelegatee the receivier of undelegation.
+     * @notice Delegate all votes `newDelegatee`. First undelegates from an existing delegate. If `newDelegatee` is zero, only undelegates.
      * @param newDelegatee the receiver of votes.
-     * @param amount the amount of votes received.
+     * @dev undefined for `delegateCount(msg.sender) > 1`
      */
-    function redelegate(
-        address oldDelegatee,
-        address newDelegatee,
-        uint256 amount
-    ) public virtual {
-        _undelegate(msg.sender, oldDelegatee, amount);
-        _delegate(msg.sender, newDelegatee, amount);
+    function delegate(address newDelegatee) external virtual {
```

Comment:
What about the case where `newDelegate == address(0)` and `count == 0`?  I think `delegateCount()` can return a `0`

This function wouldn't do anything, but maybe worthwhile throwing an error in that case to highlight to the user that the input args as not as expected/not valid